ext/lib/gui/lvgl/lvgl_mem_kernel.c

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Separate issue: Quite a few of this files placed in ext/, would seem to not be copied directly from an external project, but be actual written by you directly for integrating it with Zephyr?

Reply:
Correct they are part of a pending PR #6826 and this PR sample code depends on it.